src/profile/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Profile Diffs</title>
+    <link rel="stylesheet" href="style.css" />
+    <script type="module" src="script.js" defer></script>
+  </head>
+  <body>
+    <div id="cover-spin"></div>
+    <nav>
+      <h1>PROFILE DIFFERNCES</h1>
+    </nav>
+
+    <p id="loader">Loading...</p>
+
+    <footer id="footer">
+      <p class="info-repo">
+        The contents of this website are deployed from this
```

Comment:
![image](https://user-images.githubusercontent.com/45538392/156773247-5dec3378-b203-451b-ba1f-e22c0d4493b6.png)
![image](https://user-images.githubusercontent.com/45538392/156773270-5f678361-c354-49c1-99f3-5237dd81638c.png)

Its good to have a consistent style across different pages of same site

Reply:
Fixed.

Reply:
Fixed.